spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample/src/main/java/com/example/TradeRepository.java

```diff
@@ -32,21 +32,22 @@
  * @author Chengyuan Zhao
  */
 @RepositoryRestResource(collectionResourceRel = "trades", path = "trades")
-public interface TradeRepository extends SpannerRepository<Trade, Key> {
+public interface TradeRepository extends SpannerRepository<com.example.Trade, Key> {
```

Comment:
Try removing the fully qualified path "com.example" everywhere in this class. I think this got added while IntelliJ was still confused about where the entity moved to.